zulipterminal/core.py

```diff
@@ -253,6 +254,23 @@ def show_edit_history(
             'area:msg'
         )
 
+    def view_in_browser(self, message_id: int) -> None:
+        url = '{}#narrow/near/{}'.format(self.model.server_url, message_id)
+
+        # Don't try to open web browser if running without a GUI
+        if (LINUX and not os.environ.get('DISPLAY')
+                and os.environ.get('TERM')):
+            self.view.set_footer_text(
+                'No DISPLAY environment variable specified. This could '
+                'likely mean your system is running without a GUI.', 3)
+            return
+        # Set a footer text if no runnable browser is located
+        try:
+            webbrowser.get()
```

Comment:
What's the requirement for this particular statement? We're not using the controller being returned.

Reply:
This statement is particularly only for Error handling, in case no browser is found in the system.
Yes, we don't have any use for the returned controller, but if the module could not find any browser(GUI/text-based) then this statement will return a `webbrowser.Error` and we can use the `try-except` block to catch it and display an appropriate error message.

Reply:
Makes sense!